efi/secureboot_policy.go

```diff
@@ -210,10 +123,24 @@ func buildSignatureDbUpdateList(keystores []string) ([]*secureBootDbUpdate, erro
 			if err != nil {
 				continue
 			}
-			if strings.HasPrefix("..", rel) {
+			if strings.HasPrefix(rel, "..") {
 				continue
 			}
-			updates = append(updates, &secureBootDbUpdate{db: filepath.Dir(rel), path: line})
+
+			var name efi.VariableDescriptor
+			switch filepath.Dir(rel) {
+			case "PK":
+				name = PK
+			case "KEK":
+				name = KEK
+			case "db":
+				name = Db
+			case "dbx":
+				name = Dbx
```

Comment:
only this case is tested

Reply:
As this code is eventually going to be replaced and this specific functionality isn't used at the moment, it might be easier for now to just remove this entirely rather than creating new testdata and writing new tests for it.